src/graphql/resolvers/queries/sessions.js

```diff
@@ -10,5 +12,9 @@ export const fieldResolvers = {
       dlog('me called');
       return {};
     },
+    session: (_, { slug }, { dataSources: { firestore, logger } }) => {
+      dlog('session by slug called "%o"', slug);
```

Comment:
should be %s for string not an object

Reply:
Didn't think it actually mattered for string. Updated